scripts/generic/README.md

```diff
@@ -4,31 +4,33 @@ This is a basic explanation on how to utilize this section of the repo to easily
 
 ## Usage
```

Comment:
I made some "improvements" (subjective) to the README to address some confusion I had while following the instructions.